zwave_js/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 0.3.1
+
+### Features
+
+- Add-On: Provide access to Z-Wave JS cache files for debugging in `/addon_configs/core_zwave_js/cache`

```

Comment:
The cache files aren't documented. Should we do that?

Reply:
another thing to help with troubleshooting. We can remove it if desired

Reply:
I mean should we document it?